src/MerQure.RbMQ/SharedConnection.cs

```diff
@@ -25,13 +26,39 @@ private static IConnection GetRabbitMqConnection(MerQureConnection rabbitMqConne
         {
             ConnectionFactory connectionFactory = new ConnectionFactory
             {
-                Uri = new Uri(rabbitMqConnection.ConnectionString),
                 AutomaticRecoveryEnabled = rabbitMqConnection.AutomaticRecoveryEnabled,
                 TopologyRecoveryEnabled = rabbitMqConnection.TopologyRecoveryEnabled,
                 ClientProvidedName = rabbitMqConnection.FriendlyName,
                 RequestedChannelMax = rabbitMqConnection.RequestedChannelMax
             };
+
+            if (rabbitMqConnection.ConnectionCluster != null && !string.IsNullOrEmpty(rabbitMqConnection.ConnectionString))
```

Comment:
```suggestion
            if (rabbitMqConnection.ConnectionCluster == null && string.IsNullOrEmpty(rabbitMqConnection.ConnectionString))
```
plutôt le contraire non ?

Reply:
Pour moi, il faut les deux. J'ai adapté, je te laisse valider 😉

Reply:
c'est mieux :)